.changes/unreleased/Added-20240629-000342.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Added
+body: 'cli: allow calling core functions directly'
```

Comment:
:pray: could we get another line here? A small example, or even just a redirection to run `dagger core --help`.

Reply:
Added in https://github.com/dagger/dagger/pull/7310/commits/1d36e77d177a6c7d4614feb490dddd7c2f859ea2.